ansible/roles/docker/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
 - apt: name=apparmor
   tags: 
     - build_ami
-- apt: name=docker-engine=1.7.0-0~trusty
+- apt: name=docker-engine=1.9.1-0~trusty
```

Comment:
Docker 1.11 is out afaik.  I'm wondering if the round-robin DNS features in Docker 1.11 mean that ECS will schedule more than one web worker on a minion?

I figured that my update to docker engine 1.9.1 might raise questions.  I think we got rid of the cloudwatch/kinesis logging in lieu of shipping logs directly to sumologic.